File: JAVA SCRIPT/16.js
//16. Check if all the elements in arr = [3, 5, 9, 1, 7] are positive numbers, and print true or false.
var arr=[3,5,9,1,7];
var p=0;
for(i=0;i<arr.length;i++){
    if(arr[i]>0){
        p++;
    }
}
if(p==arr.length){
    console.log("true");
}
else{
    console.log("false");
}

File: C LANGUAGE/16.c
//16. Check if all the elements in arr = [3, 5, 9, 1,7] are positive numbers, and print true or false.

#include<stdio.h>
int main(){
	int arr[]={3,-5,9,1,7};
	int i;
	int length=sizeof(arr)/sizeof(arr[0]);
	int count=0;
	for(i=0;i<length;i++){
		if(arr[i]>0){
			count++;
		}
	}
	if(count==length){
		printf("true");
	}
	else{
		printf("false");
	}
}
